.env.example

```diff
@@ -4,10 +4,14 @@ SERVER_CRT=test/localhost.crt
 SERVER_KEY=test/localhost.key
 SERVER_MAX_PEERS=128
 SERVER_DATABASE_BACKEND=json
-DEBUG=true
+# Only required if backend is "json"
+SERVER_JSON_BACKEND_FILE="data/users.json"
```

Comment:
I'm still learning path stuff within godot. Should this instead be `"res://data/users.json"` or does that not matter in this case?

Reply:
You're right. "res://" is used to point to the current project directory. Thus it's a safer option.